framework/src/util/Utils.cpp

```diff
@@ -164,12 +213,8 @@ bool Exists(const std::string& path)
     else throw std::invalid_argument(GetLastErrorStr());
   }
 #else
-  DWORD attr(::GetFileAttributes(path.c_str()));
```

Comment:
The reason for change is PathFileExistsW supports UNC paths and GetFileAttributes does not. I was unaware of the performance difference. This particular change is not really needed to support unicode so I am reverting it.